codecparsers/bitReader.cpp

```diff
@@ -64,6 +64,13 @@ void BitReader::loadDataToCache(uint32_t nbytes)
     m_bitsInCache = nbytes << 3;
 }
 
+inline void BitReader::reload()
+{
+    uint32_t remainingBytes = m_size - m_loadBytes;
```

Comment:
I know it. What I mean is that we should make sure each independent function's robustness. m_size and m_loadBytes are changed by  the other functions, and this reload function doesn't make sure  m_size > m_loadBytes.